src/components/Form/Form.tsx

```diff
@@ -49,12 +52,15 @@ export default class Form extends React.PureComponent<Props> {
       name,
       noValidate,
       target,
+      polaris: {intl},
     } = this.props;
     const autoCompleteInputs = normalizeAutoComplete(autoComplete);
 
     const submitMarkup = implicitSubmit ? (
       <VisuallyHidden>
-        <button type="submit" aria-hidden="true" />
+        <button type="submit" aria-hidden="true">
+          {intl.translate('Polaris.Common.submit')}
```

Comment:
If the button is aria hidden, it won’t be seen by screen reader users anyway, right? Is this intended, or should we be removing the `aria-hidden`?

Reply:
Thats true, however buttons always need content inside of them. This is probably a false positive but I would prefer the button to have content instead of being empty.

Reply:
Although I’d expect tests to pass with empty content and `aria-hidden="true"`, I’m cool with this fix.